gooddata-java-model/src/main/java/com/gooddata/sdk/model/featureflag/FeatureFlags.java

```diff
@@ -5,11 +5,18 @@
  */
 package com.gooddata.sdk.model.featureflag;
 
-import com.fasterxml.jackson.annotation.*;
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
```

Comment:
Plz use default IDEA setting for star imports in this project - i.e. 5 (3 for static)

Reply:
OMG why should I always switch between different import settings when being in this company... :(